units/modprobe@.service

```diff
@@ -33,3 +33,6 @@ StartLimitIntervalSec=0
 [Service]
 Type=oneshot
 ExecStart=-/sbin/modprobe -abq %i
+
+[Install]
+WantedBy=sysinit.target
```

Comment:
I'm not convinced. We have `module-load.d/` specifically for such use case. This OTOH is about loading on-demand, when a unit actually pulls it in and requires ordering.